BoxOffice/Model/CellMode.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CellMode.swift
+//  BoxOffice
+//
+//  Created by Rhode, Rilla on 2023/04/11.
+//
+
+enum CellMode: Codable {
+    static let identifier = "CellMode"
+    
+    case List
+    case Icon
```

Comment:
enum의 case의 네이밍은 CamelCase로 해야합니다.
```suggestion
    case list
    case icon
```

Reply:
이런 기초적인걸 놓치다니 아직 갈길이 멉니다,,,🥲